content/docs/user-guide/networking/autopackets.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Networking Auto-packets
+description: A reference for defining Open 3D Engine networking packets through auto-packets.
+linktitle: Auto-packets
+---
+
+*Auto-packets* provide a convenient way to define network packets used to communicate data between endpoints in a `AzNetworking` session. Using the [AzAutoGen](/docs/user-guide/engine/autogen) system, auto-packet files found inside of your project are processed during builds to create C++ classes and handlers for packets with defined payloads.
+
+In order to enable auto-packet builds for your project, you must add AZ::AzNetworking as a build dependency of your project.
+
+## Auto-packet file structure
+
+Auto-packets are defined in XML files, placed in the `Code\Source\Autogen` directory of the relevant project or Gem.
+
+### PacketGroup attributes
+
+The top-level element of an auto-packet definition is a `PacketGroup`. Packet groups associate different types of individual packet definitions together. Packets defined as part of a packet group are placed into a namespace.
+
+| Property | Description | Type |
+|---|---|---|---|
+| Name | The namespace for generated auto-packets. | A valid C++ namespace identifier |
+| PacketStart | The value to start sequencing packet type identifiers for this packet group at. To avoid conflicts with the core AzNetworking framework, use `CorePackets::PacketType::Max`. | `const int32` |
+
+### Packet attributes
+
+The `Packet` tag defines a new packet.
+
+| Property | Description | Type |
+|---|---|---|---|
+| Name | The class name of the generated autopacket. | A valid C++ class identifier. |
+| Desc | The description of the generated autopacket. | `string` |
+| HandshakePacket | If the packet is part of a connection handshake. | `bool` |
+{{< note >}}
+Packets with `HandshakePacket ` set as true will only be processed if the user-implemented `IsHandshakeComplete` function for the packet returns `false`.
+{{< /note >}}
+
+### Member attributes
+
+The `Member` tag defines data on the packet. This is the primary mechanism for defining a packet's payload.
+
+| Property | Description | Type |
+|---|---|---|---|
+| Name | The name of the packet member. | A valid C++ variable name. |
```

Comment:
Do any of these fields have size/limits on the type?

Reply:
Nope, these aren't specialized types so there's nothing explicit. Granted larger counts on containers will yield larger packets.